src/features/dashboard/dashboard-page.js

```diff
@@ -1,68 +1,124 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useOutletContext, useNavigate } from 'react-router';
 
-import { Container } from '@mui/material';
+import { Container, Button } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
 
+import CustomTable from 'components/table/table';
 import DateRangeFilter from 'features/filters/date-range-filter';
 import DashboardCard from 'components/DashboardCard';
 import IncomesGraphCard from 'components/IncomesGraphCard';
 import InvoiceSummaryGraphCard from 'components/InvoiceSummaryGraphCard';
+import { ROUTES } from 'common/constants';
+import { getSummary } from './dashboard-slice';
 import styles from './dashboard-page.module.scss';
 
 const PAGE_TITLE = 'Dashboard';
 const Dashboard = () => {
   const user = useSelector((state) => state.user);
+  let summaryData = useSelector((state) => state.summary);
   const [setHeaderTitle] = useOutletContext();
-
-  useEffect(() => setHeaderTitle(PAGE_TITLE), [setHeaderTitle]);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
 
   const filtersConfig = {
-    rangeValue: '12months',
+    rangeValue: '12',
     rangeOptions: [
-      { value: '12months', label: 'Last 12 months' },
-      { value: '6months', label: 'Last 6 months' },
-      { value: '3months', label: 'Last 3 months' },
-      { value: '1month', label: 'Last month' },
+      { value: '12', label: 'Last 12 months' },
+      { value: '6', label: 'Last 6 months' },
+      { value: '3', label: 'Last 3 months' },
+      { value: '1', label: 'Last month' },
     ],
     onRangeOptionChange: () => {},
   };
 
+  useEffect(() => setHeaderTitle(PAGE_TITLE), [setHeaderTitle]);
+  useEffect(() => {
+    dispatch(getSummary(filtersConfig.rangeValue));
+  }, [dispatch]);
+
   let cardsText = [
     {
-      cardValue: 5,
+      cardValue: summaryData !== undefined ? summaryData.onGoingProjects : 0,
       cardLabel: 'Ongoing Projects',
     },
     {
-      cardValue: '$ 23456,78',
+      cardValue: summaryData !== undefined ? `$ ${summaryData.totalInvoice}` : 0,
       cardLabel: 'Total Invoices',
     },
     {
-      cardValue: '$ 567,99',
+      cardValue: summaryData !== undefined ? `$ ${summaryData.pendingInvoice}` : 0,
       cardLabel: 'Pending Invoices',
     },
     {
-      cardValue: '$ 909,30',
+      cardValue: summaryData !== undefined ? `$ ${summaryData.totalOverdue}` : 0,
       cardLabel: 'Total Overdue',
     },
   ];
 
+  const projectColumns = [
+    { field: 'projectNames', displayName: 'Projects', width: 30 },
+    { field: 'view', displayName: 'View all', width: 15 },
+  ];
+
+  const invoicesColumns = [
+    { field: 'invoice', displayName: 'Latest Invoices', width: 30 },
+    { field: 'view', displayName: 'View all', width: 15 },
+  ];
+
+  //const projectsData = useSelector((state) => state.projects);
+  const projectsData = [{ projectNames: 'Web Page creation\nClient Name 1', view: 'END DATE\n2022-07-21' },
```

Comment:
@KariGarcia This is the way the field comes "projectNames" have 2 texts separated by a \n